bookshelf/config/application.rb

```diff
@@ -11,26 +11,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require File.expand_path("../boot", __FILE__)
+require_relative 'boot'
 
 require "rails"
-
 # Pick the frameworks you want:
 require "active_model/railtie"
```

Comment:
It looks like we do need:
```ruby
require "active_job/railtie"
```
in the production environment. (I'm not sure why, since we're not using it, but the app crashes without it.)

Reply:
Added.